### ISSUE TITLE
no-unused-styles does not support React Fragment shorthand

### ISSUE THREAD

**Issue by alvinthen** [Intellicode/eslint-plugin-react-native#258]
As per title, when using `<></>`, eslint does not report any errors.

However, `<React.Fragment></React.Fragment>` does work

```jsx
import React from 'react';
import { View, StyleSheet } from 'react-native';

export default function Tester() {
  return (
    <>
      <View style={styles.used} />
    </>
  );
}

const styles = StyleSheet.create({
  unused: {},
  used: {},
});
```
